packages/cadl-ranch-specs/http/azure/client-generator-core/access/main.tsp

```diff
@@ -0,0 +1,191 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+
+@doc("Test for internal decorator.")
+@supportedBy("dpg")
+@scenarioService("/azure/client-generator-core/access")
+namespace _Specs_.Azure.ClientGenerator.Core.Access;
+
+@route("/public")
+@global.Azure.ClientGenerator.Core.operationGroup
+@scenario
+@scenarioDoc("""
+This scenario contains public operations. It should be generated and exported.
+Expected query parameter: name=<any string>
+Expected response body:
+```json
+{
+  "name": <any string>
+}
+```
+""")
+namespace Public {
```

Comment:
Why namespace, not interface? May need `@operationGroup`

Also you may want to avoid using possible reserved keyword like `public`/`private`

Reply:
I just want to group relative model and operation together. Any problem for the namespace?

Reply:
Likely won't affect Java (but all operations and models would get to a single client, without `@operationGroup`, so the test and test client won't be grouped).

Not sure about other languages.

Reply:
I believe it is valid spec, right? If so, other languages should support.

Reply:
Sure, we will require all language approve then.